static/js/components/ErrorMessage_test.js

```diff
@@ -144,7 +144,7 @@ describe("ErrorInfo", () => {
     });
 
     describe('profile page', () => {
-      it('shows an error when profile get has errored', () => {
+      it('shows an error when profile get has errored instead of leaving the spinner up', () => {
```

Comment:
hm so it looks like this test was actually a duplicate of the test above it in `ProfilePage_test.js`. I'm going to just delete it.